### ISSUE TITLE
Release schedule (v1.22.0)

### ISSUE THREAD

**Issue by wenzhixin** [wenzhixin/bootstrap-table#6731]
### Description

Plan date: 2023-05-01
Plan PR list:

**Comment by marceloverdijk** [comment]
Hi @wenzhixin what is the status for `1.22.0` ?

Would love to see #6796 (has PR) and also especially #6633 (PR assigned to @UtechtDustin) to be be included.

**Comment by UtechtDustin** [comment]
@wenzhixin the planned date (2023-05-01) is over one month ago.
Any news on the release ?

**Comment by wenzhixin** [comment]
Mainly because there are not many changes.

**Comment by UtechtDustin** [comment]
I don't think we should only release a new version after x changes.
If we done something we should release it, ppl will wait for the change(s) like @marceloverdijk.
Also there are 104 commits if i'm correct :)

**Comment by marceloverdijk** [comment]
Yes, I would love to see more continuous releases as well — even with few updates. The waiting is now a bit long.

Note the is [one](https://github.com/wenzhixin/bootstrap-table/pull/6829) pending PR I could really use.
And I try to make another one this weekend for https://github.com/wenzhixin/bootstrap-table/issues/6798 (bullet 2: `column.switchableLabel`).

**Comment by marceloverdijk** [comment]
Here you go, I've just submitted 2 PR's for #6798 adding the `switchableLabel` column option.
- https://github.com/wenzhixin/bootstrap-table-examples/pull/483
- https://github.com/wenzhixin/bootstrap-table/pull/6832

Hope to see the new release soon ;-)

**Comment by wenzhixin** [comment]
@marceloverdijk, great job!

**Comment by marceloverdijk** [comment]
Would it be possible to include PR https://github.com/wenzhixin/bootstrap-table/pull/6829 ? That one is quite useful for me.

**Comment by UtechtDustin** [comment]
> Would it be possible to include PR #6829 ? That one is quite useful for me.

Sure, merged it :)

**Comment by wenzhixin** [comment]
I will release a new version today.
